references/V1_OutboundCallback.yml

```diff
@@ -175,6 +175,45 @@ definitions:
       - spaceName
       - memberIds
       - time
+  MessageEditedBody:
+    type: object
+    description: |
+      Notifies that an administrator has updated the contents of a message in a space.

```

Comment:
How about removing why/who the message has been edited and just leave it as:

"Notifies the subscribed app that a message has been edited"

Reply:
Addressed in commit 76554ff